app/models/floating_ip/operations.rb

```diff
@@ -0,0 +1,19 @@
+module FloatingIp::Operations
+  extend ActiveSupport::Concern
+
+  def update_floating_ip(options = {})
+    raw_update
```

Comment:
@andyvesel Looking through the code base there is some inconsistency with the naming of the "raw_" methods, but I would expect the "raw_" method name to be `raw_update_floating_ip`.  And you need to pass the options hash.

Same for the `delete_floating_ip` method.  I would expect the "raw_" method to be `raw_delete_floating_ip`.

cc @Fryguy @blomquisg